src/Dapr.Actors/Runtime/ActorService.cs

```diff
@@ -21,15 +22,18 @@ public class ActorService : IActorService
         /// <param name="actorTypeInfo">The type information of the Actor.</param>
         /// <param name="loggerFactory">The logger factory.</param>
         /// <param name="actorFactory">The factory method to create Actor objects.</param>
+        /// <param name="jsonSerializerOptions">The <see cref="JsonSerializerOptions"/> to use when actor state persistence and message deserialization</param>
```

Comment:
```suggestion
        /// <param name="jsonSerializerOptions">The <see cref="JsonSerializerOptions"/> to use for actor state persistence and message deserialization.</param>
```

Reply:
Where does this get used for message deserialization? I only see it used for state persistence in this PR.

Reply:
After rebasing to master I have removed this and moved the JsonSerializerOptions for the run time over to the `ActorRuntimeOptions` class. This is similar to the ActorClient code that has the JsonSerializerOptions in the ActorProxyOptions. I think this helps simplify where the user should be customizing the JsonSerializerOptions.